kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -1,11 +1,20 @@
-{% extends "kolibri/base.html" %}
```

Comment:
Mainly because I may have to do some work on the `template_tags` and wanted to consolidate all logic in one place until it becomes cleared what's definitely going to be in the Django template. I'm also still toying with the idea of parameterizing a single shared `base.html` template, and passing as arguments things like sync/async modules to load and initial data to bootstrap.